sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -12,6 +12,7 @@
 #include <linux/bits.h>
 #include <linux/types.h>
 #include <sound/soc.h>
+#include "sof_sdw_hdmi.h"
```

Comment:
let's rename this as sof_hdmi_common.h, there's nothing related to sdw in there

Reply:
done